frameworks/Erlang/elli/bash_profile.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
We're no longer using bash_profile.sh. exports need to be moved to setup.sh/install.sh